From: unknown
Upstream: no
References: none

Update from K&R to ANSI prototype.   This fix should go upstream.

Index: elfutils-0.155/libebl/eblopenbackend.c
===================================================================
--- elfutils-0.155.orig/libebl/eblopenbackend.c
+++ elfutils-0.155/libebl/eblopenbackend.c
@@ -249,10 +249,7 @@ fill_defaults (Ebl *result)
 
 /* Find an appropriate backend for the file associated with ELF.  */
 static Ebl *
-openbackend (elf, emulation, machine)
-     Elf *elf;
-     const char *emulation;
-     GElf_Half machine;
+openbackend (Elf *elf, const char *emulation, GElf_Half machine)
 {
   Ebl *result;
   size_t cnt;
Index: elfutils-0.155/lib/dynamicsizehash.c
===================================================================
--- elfutils-0.155.orig/lib/dynamicsizehash.c
+++ elfutils-0.155/lib/dynamicsizehash.c
@@ -44,10 +44,7 @@
 
 
 static size_t
-lookup (htab, hval, val)
-     NAME *htab;
-     HASHTYPE hval;
-     TYPE val __attribute__ ((unused));
+lookup (NAME *htab, HASHTYPE hval, TYPE val __attribute__((unused)))
 {
   /* First hash function: simply take the modul but prevent zero.  */
   size_t idx = 1 + hval % htab->size;
